### ISSUE TITLE
Plex User Importer failing

### ISSUE THREAD

**Issue by war3zlod3r** [Ombi-app/Ombi#5004]
### Summary

The messages appears saying import started and then very quickly says finished, I can't find any indication of the attempt in the logs. Not sure whats going on but the new plex users are not importing into Ombi.

### Ombi Version

4.39.1

### What platform(s) does this occur on?

Docker

### What database are you using?

SQLite (Default)

### Relevant log output

```shell
The log doesn't seem to show an output error or even the attempt to import.
```

**Comment by eskemojoe007** [comment]
I am receiving the exact same issue using mariaDB.  Not DB log issues, no ombi log issues, no linuxserver docker log issues to be found.  Connection to plex is working great and syncs content, just can't add a user via the importer.

**Comment by eskemojoe007** [comment]
Looking through the change logs, I was able to see that they fixed this already in 4.39.4 in issue https://github.com/Ombi-app/Ombi/issues/4989.  Switching my docker to the [4.43.4-development](https://hub.docker.com/layers/linuxserver/ombi/4.43.4-development/images/sha256-2e1183dbc1995762ed88f5ded32d19c33a4f5598f4ccaf03d9accdce8c0afcec?context=explore) completely solved the issue for me.  Recommend upgrading manually, or waiting for the next bigger release.

**Comment by ofirkarin** [comment]
I have a similar issue. I granted access to my libraries to a new user, the user appears on my friends list on Plex but they are not imported to Ombi. After running. the importer (indicating the import was successful) the following line appears in the logs:

> Start processing HTTP request GET "https://plex.tv/pms/friends/all/"

The request is then answered and when viewing the debug level logs the new user really does not appear in the XML returned from Plex. I then generated a token and manually accessed "https://plex.tv/pms/friends/all?X-Plex-Token={token}". All other users appear there but the new user doesn't.

I gave it a few days but the XML is still not up to date. After a but of searching around I saw that the new user also does not appear in "https://plex.tv/pms/friends/requestes" or in "https://plex.tv/pms/friends/requests". However, they do appear (along with all the rest) in "https://plex.tv/pms/users". I'm not sure if that's relevant, just seemed odd but I don't know the purpose of the second address. I'll mention that some of my users are home users and some aren't. In the XML in which the new user appears they have the same attributes as other non-home users that do appear in the "friends/all" XML.

Apologies if the comment is out of place or if I misused the conventions for bug reports, I wasn't sure if I should open a new issue.
